example/advanced/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  bitcoind:
+    image: zquestz/bitcoin-abc
+    # specify additional command line arguments here
+    command: bitcoind
+    healthcheck:
+      test: ["CMD", "/entrypoint.sh", "bitcoin-cli", "getinfo"]
+    ports:
+      # change to 18333 if running testnet
+      - "8333:8333"
+      # JSON-RPC
+      - "8332:8332"
+    volumes:
+      # using a custom path for data dir
+      - /mnt/volume-sfo2-03/data:/data
```

Comment:
This might be cleaner as having a data dir in this directory that it mounts, much like the bitcoin.conf.